### ISSUE TITLE
Linux error: "ELF file OS ABI invalid"

### ISSUE THREAD

**Issue by kriegaex** [warmuuh/libsass-maven-plugin#27]
I see the following error on our Jenkins server (CentOS release 5.11):

```
An API incompatibility was encountered while executing
com.github.warmuuh:libsass-maven-plugin:0.1.7-libsass_3.2.4:compile:
  java.lang.UnsatisfiedLinkError:
    /tmp/jna-3506402/jna4265422068358906463.tmp: ELF file OS ABI invalid
```

I see that _/tmp/jna-3506402/_ is an empty directory after the build was aborted. Some more info:

```
$ uname -s -r -v -p -i -o
Linux 2.6.18-407.el5 #1 SMP Wed Nov 11 08:12:41 EST 2015 x86_64 x86_64 GNU/Linux
```

**Comment by warmuuh** [comment]
i switched the native bindings to an external library, can you try again using the current version (0.2.0-libsass_3.3.2), it might fix the issue...   its java8-only, might be an issue though..

**Comment by kriegaex** [comment]
Thanks for publishing this new version today. I can acknowledge that the problem is fixed now even on our old Linux 2.6 Jenkins server. Good job! :-)

**Comment by aaitmouloud** [comment]
Any Java 7 version running this fix?
